tests/integration/redshift.spec.ts

```diff
@@ -14,7 +14,6 @@ describe("@dataform/integration/redshift", () => {
     });
 
     expect(compiledGraph.graphErrors.compilationErrors).to.eql([]);
-    expect(compiledGraph.graphErrors.validationErrors).to.eql([]);
```

Comment:
let's keep this line for now! it's important to assert that, while we still ever put stuff in that field, the integration tests never have validation errors.

Reply:
Oh sorry my bad. I've been in parallel working on #247 and this change actually belongs there.